app/steps/ui/executors/currentnamereason/index.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const ValidationStep = require('app/core/steps/ValidationStep');
+const {findIndex, get} = require('lodash');
+const ExecutorsWrapper = require('app/wrappers/Executors');
+
+module.exports = class ExecutorCurrentNameReason extends ValidationStep {
```

Comment:
Can we export the class at the bottom of the file, see `app/steps/ui/executors/mentalcapacity/index.js` for an example.